examples/chess/README.md

```diff
@@ -52,24 +49,21 @@ The [guest code] checks that applying the move to the initial board state is leg
 - For a video walk-through of this example, check out this [excerpt from our workshop at ZK HACK III].
 - For more information about building, running, and testing zkVM applications, see our [developer docs].
 
-[`src/main.rs`]: src/main.rs
-[`methods/guest/src/main.rs`]: methods/guest/src/main.rs
-[host]: https://dev.risczero.com/terminology#host
+[Bonsai application]: https://dev.bonsai.xyz
+[developer docs]: https://dev.risczero.com
+[examples guide]: https://dev.risczero.com/api/zkvm/examples/#running-the-examples
+[excerpt from our workshop at ZK HACK III]: https://www.youtube.com/watch?v=vxqxRiTXGBI&list=PLcPzhUaCxlCgig7ofeARMPwQ8vbuD6hC5&index=9
 [executes]: https://dev.risczero.com/terminology#execute
+[guest code]: methods/guest/src/main.rs
 [guest program]: https://dev.risczero.com/terminology#guest-program
+[host code]: methods/guest/src/main.rs
 [host program]: https://dev.risczero.com/terminology#host-program
+[host]: https://dev.risczero.com/terminology#host
+[journal]: https://dev.risczero.com/terminology#journal
 [proves the execution]: https://dev.risczero.com/terminology#prove
 [receipt]: https://dev.risczero.com/terminology#receipt
-[verify]: https://dev.risczero.com/terminology#verify
-[journal]: https://dev.risczero.com/terminology#journal
-[examples guide]: https://dev.risczero.com/api/zkvm/examples/#running-the-examples
-[here]: https://github.com/risc0/risc0/blob/main/examples/chess/src/main.rs#L29
```

Comment:
just for this file as an example:

- moved all links to the bottom & sorted alphabetically (VSCode has a nice select lines -> ctrl+shift+p -> sort ascending flow to help)
- removed all hard coded github self-referencing links to this repo to relative links
- de-duplicated link referenced
- pass lyche for this file

If this looks good, we can do do this kinda thing as a chore systematically throughout for .md files at least